### PR TITLE
Nav menu dropdown

### PR DESCRIPTION
**Pull request by StRhys**

### CODE REVIEW

**Review by nimshi89 — COMMENTED** [review]
Looks fine for me, specially the gov-header__logo part is a lot cleaner. I would say someone with more experience should approve this PR if that's okay

File: components/Header/_header.scss
Comment (eldeal) [inline components/Header/_header.scss]: can we remove the commented out font-size?

File: components/Header/_header.scss
Comment (eldeal) [inline components/Header/_header.scss]: remove commented out code plz

File: components/Header/index.tsx
Comment (eldeal) [inline components/Header/index.tsx]: Can we fix the file path for this to include the _ after IDS please? Just the inconsistency makes me twitch, I realize it's likely not a functional problem :)

File: components/Header/index.tsx
Comment (eldeal) [inline components/Header/index.tsx]: Also a personal preference thing, but if it doesn't breach any JS standards around file structure my instinct would be to move this hardcoded list to another file so the index.tsx focusses more directly on the crux of what it needs to do. If we refactor these items to be pulled from a data source in the future, then that complexity can happen in that other file rather than all in one? 

But if it's a bad JS pattern then ignore me!

Comment (eldeal) [reply components/Header/index.tsx]: fwiw I dont feel the same about popularItems, I'm happy for that to stay, its just the topics

**Review by eldeal — CHANGES_REQUESTED** [review]

File: components/Header/_header.scss
Comment (StRhys) [reply components/Header/_header.scss]: Removed

File: components/Header/_header.scss
Comment (StRhys) [reply components/Header/_header.scss]: Removed

File: components/Header/index.tsx
Comment (StRhys) [reply components/Header/index.tsx]: Peace of mind added

File: components/Header/index.tsx
Comment (StRhys) [reply components/Header/index.tsx]: Moved the topics list into a data folder, I should have done this before as I reuse it a couple times in some other tickets.

**Review by eldeal — DISMISSED** [review]

**Review by nimshi89 — APPROVED** [review]

**Pull request merged**
